.github/workflows/MacOsX.yaml

```diff
@@ -0,0 +1,200 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Continuous integration tests that pull requests are required to pass
+name: MacOsX
+
+# Note that by default the jobs only run on the base repository, testing pull
+# requests and merge commits. Enable GitHub Actions in your fork's repository
+# settings to also run the tests on every push to one of your branches.
+on:
+  # This is broken, so is only run manually.  When fixed, comment out this
+  # and uncomment pull_rquest: and the 3 lines starting with push:
```

Comment:
Please link to https://github.com/sxs-collaboration/spectre/issues/2397.

Typo `pull_request`, and <s>the 3 lines starting with</s>. You could also just delete the note on what to uncomment, along with all the comments below, since it's quite obvious how to re-enable by looking at the `Tests.yaml` file.